### ISSUE TITLE
Update yaml for access to S3 PRD

### ISSUE THREAD

**Issue by mvarewyck** [inbo/reporting-rshiny-grofwildjacht#367]
To use the correct bucket in the PRD environment, we have to update the shinyproxy yml file.
TODO Before #352 
```
      - id: grofwild
        container-env:
          R_CONFIG_ACTIVE: production
```

Both on private and public repository.

**Comment by mvarewyck** [comment]
FYI @stenmigerode @berthuygens

**Comment by LEDfan** [comment]
I updated the ShinyProxy config file.

**Comment by SanderDevisscher** [comment]
@LEDfan possibly you changed the config.yml on the main - branch of the shinyproxy which causes these errors (https://github.com/inbo/reporting-rshiny-grofwildjacht/pull/352#issuecomment-1363886340 & https://github.com/inbo/reporting-rshiny-grofwildjacht/pull/352#issuecomment-1363892668) . Apperently this is the branch for the UAT while the production branch is the one for the PRD. Can you correct this ?

**Comment by SanderDevisscher** [comment]
We're going to rename the `main` branch into either `UAT` or `Testing` or something similar to prevent similar mistakes.

**Comment by stenmigerode** [comment]
I have updated the branches and proposed 2 pull requests:
- 1 for removing R_CONFIG_ACTIVE in the UAT branch
- 1 for adding the R_CONFIG_ACTIVE in the PRD branch

**Comment by SanderDevisscher** [comment]
@stenmigerode has the `main` branch been renamed to `UAT` then ?

**Comment by stenmigerode** [comment]
De branch has been renamed and both uat and production branches are now producted.
A pull request is needed with at least 1 approval before merging changes onto those branches.

**Comment by SanderDevisscher** [comment]
@stenmigerode Has this been fixed in the public shinyproxy repo as well ?

**Comment by stenmigerode** [comment]
@SanderDevisscher Until now, the public shinyproxy settings have been the same between UAT and PRD. There is only one branch in that github repository. I can see that that branch was indeed also updated and the R_CONFIG_ACTIVE setting was added.

That repository was set up differently. I can see folders for DEV and PRD configs, but the application.yml file is shared between both DEV and PRD. This means we have to foresee something in the repository, and update the deploy scripts for this shiny proxy instance (which apparently also hosts other applications like WUG and waterbirds reporting).

I am currently busy with many different other urgent projects. I think I will be able to look at this in 2 or 3 weeks the earliest.

**Comment by SanderDevisscher** [comment]
@stenmigerode thanks for this update. 
@LEDfan do you think you can take a look at this within 2 or 3 weeks ? making sure you take good care not to break the other applications. Also can you give an update stating the time you would need, so I can change our planning ? 

@berthuygens with this new info I think it is best to postpone the testing of the s3 bucket - branch until this issue has been fixed in the public shinyproxy repo.

**Comment by LEDfan** [comment]
Hi @SanderDevisscher , I can revert the commit I did to the public shinyproxy version in order to make the public app working again. (but someone else will have to deploy it) However, I don't have access to your AWS accounts and I don't know much about the internal setup, therefore I'm not able to change the setup into a UAT and PROD setup.

**Comment by SanderDevisscher** [comment]
@LEDfan I suggest you indeed start with reverting the changes to the public shinyproxy so we can at least test #346 & #356.

**Comment by LEDfan** [comment]
@SanderDevisscher I just reverted the change.

**Comment by SanderDevisscher** [comment]
I consider this issue fixed by #352
